config.md

```diff
@@ -371,10 +371,19 @@ Possible values: `None` (default), `Strict`.
 
 ## Completion
 
+Configures code completion features. Sample block (default):
+
+```yaml
+Completion:
+    AllScopes: Yes
+    ArgumentLists: FullPlaceholders
+    HeaderInsertion: Never
```

Comment:
I assume by "sample block (default)" you mean that you want to list the defaults here.

The default for `HeaderInsertion` is `IWYU`.

Reply:
Ah whoops! Fixed it.